src/ui/public/courier/fetch/request/segmented.js

```diff
@@ -100,7 +107,7 @@ define(function (require) {
     };
 
     SegmentedReq.prototype.isIncomplete = function () {
-      return this._queue.length > 0;
+      return !this._queueCreated || this._queue.length > 0;
```

Comment:
might read better as `this._queueCreated &&`. I missed the `!` at first and thought this was backwards